sshdconfig/tree-sitter-ssh-server-config/bindings/rust/build.rs

```diff
@@ -0,0 +1,21 @@
+fn main() {
```

Comment:
We don't want to check in the bindings, but have them generated each time in case we update to newer versions of tree-sitter, see https://github.com/PowerShell/DSC/blob/main/tree-sitter-dscexpression/tree-sitter.json

Reply:
oops, didn't mean to commit those!